### PR TITLE
Changes to override default aws s3 path

### PR DESCRIPTION
**Pull request by abhivijay89**

### CODE REVIEW

File: appender-core/src/main/java/com/van/logging/aws/S3Configuration.java [hunk 1]
```diff
@@ -161,8 +160,14 @@ static Region resolveRegion(String str) {
     }
 
     public String toString() {
-        return String.format("S3 configuration (%s:%s in region %s; compressed: %s)",
-            this.bucket, this.path, this.region, this.compressionEnabled
-        );
+        if (this.path != null && !this.path.isEmpty()) {
```
Comment (bluedenim) [inline appender-core/src/main/java/com/van/logging/aws/S3Configuration.java:163]: Try using `StringUtils.isTruthy(this.path)`?

https://github.com/bluedenim/log4j-s3-search/blob/master/appender-core/src/main/java/com/van/logging/utils/StringUtils.java#L15

File: appender-core/src/main/java/com/van/logging/aws/S3PublishHelper.java [hunk 2]
```diff
@@ -46,10 +47,14 @@ public S3PublishHelper(S3Configuration s3, boolean verbose) {
 
         this.bucket = s3.getBucket().toLowerCase();
         String path = s3.getPath();
-        if (!path.endsWith("/")) {
-            this.path = path + "/";
+        if (path != null && !path.isEmpty()) {
```
Comment (bluedenim) [inline appender-core/src/main/java/com/van/logging/aws/S3PublishHelper.java:50]: Also try using `StringUtils.isTruthy(path)` here.

File: appender-core/src/main/java/com/van/logging/aws/S3Configuration.java (hunk 1 above)
Comment (abhivijay89) [reply appender-core/src/main/java/com/van/logging/aws/S3Configuration.java]: Sure. Incorporated the change

File: appender-core/src/main/java/com/van/logging/aws/S3PublishHelper.java (hunk 2 above)
Comment (abhivijay89) [reply appender-core/src/main/java/com/van/logging/aws/S3PublishHelper.java]: Incorporated the change

**Pull request merged**